### DISCUSSION TITLE
[V5] Lucid Model Relationships that included in AdonisJS 4.1

### DISCUSSION THREAD

**Discussion by remidosol** [adonisjs/core#1637] [Help] [answered]
Hello folks 👋 

I'm wondering that will Lucid Relationships that AdonisJS 4.1 contains been to be added to AdonisJS 5?
I need very much to BelongsToMany.

Or can I write this relationship? If I can write, what am I suppose to do, what resourses should I use to?

@RomainLanz 
@thetutlage

**Comment by thetutlage** [comment, marked as answer]
All the relationships already exists in v5. The belongsToMany has been renamed to many to many

**Reply by remidosol** [reply]
Thanks. I'd might not been understanded it when I was reading the documentation.
